src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/manageusers/ManageUsersApiClient.kt

```diff
@@ -7,14 +7,14 @@ import org.springframework.web.reactive.function.client.WebClientResponseExcepti
 
 @Component
 class ManageUsersApiClient(@Qualifier("manageUsersApiWebClient") private val manageUsersApiClient: WebClient) {
-  fun getUserDetails(username: String): UserDetailsDto? {
+  fun getUserDetails(username: String): UserDetailsDto {
```

Comment:
Minor change to the signature here as this function will either return a UserDetailsDto or throw a ManageUsersApiException
I dont think there is a case / code path where it can return null?